lib/active_record/connection_adapters/oracle_enhanced/jdbc_connection.rb

```diff
@@ -457,7 +457,7 @@ def write_lob(lob, value, is_binary = false)
         end
 
         # To allow private method called from `JDBCConnection`
-        def describe(name)
+        def describe(name, supports_longer_identifier)
```

Comment:
Do you mean `supports_longer_identifier?`

Reply:
Yes. It takes the result of `ActiveRecord::ConnectionAdapters::OracleEnhancedAdapter#supports_longer_identifier?` as argument. Unfortunately no better name has come up yet.